finance/funding_requests.md

```diff
@@ -1,82 +1,68 @@
-The Astropy project wants to make life easy for all the volunteers and people working towards our shared goal; on the other hand, we also need to make sure we spend grant money in a fiscally responsible way and as allowed by donors or funding agencies. This policy thus aims to strike a balance between as little bureaucracy as possible while also ensuring that money is spend responsibly and that Astropy has all the documentation that are required for any of the grants given to Astropy.
-
-
-
-Funding requests can be made to the Interim Astropy Finance Committee.  The following provides a description for how to obtain funding for different types of activities. 
-If you have questions on how to apply this process, please email finance@astropy.org .
+While the Astropy project wants to make life easy for all the volunteers and people working towards our shared goal we need to make sure we spend grant money in a fiscally responsible way. This policy thus aims to strike a balance between as little bureaucracy as possible and ensuring that Astropy has all the documentation required by funders supporting Astropy.
 
+The following provides a description for how to obtain funding for different types of activities. All funding requests should be made to the Astropy Finance Committee. If you have questions on how to apply this process, please email finance@astropy.org.
```

Comment:
or open an issue.

Reply:
I agree with @kelle here!  Should be added here explicitly as an option.